src/main/gen/movie/GetAllShows.kt

```diff
@@ -21,7 +21,8 @@ public class GetAllShowsRowMapper : RowMapper<GetAllShowsResult> {
   public override fun map(rs: ResultSet): GetAllShowsResult = GetAllShowsResult(
   id = rs.getObject("id") as kotlin.Int,
     startTime = rs.getObject("start_time") as java.sql.Timestamp,
-    movieId = rs.getObject("movie_id") as kotlin.Int)
+    movieId = rs.getObject("movie_id") as kotlin.Int,
+    price = rs.getObject("price") as kotlin.Int)
```

Comment:
Please write the relevant sql query to generate this file. 
Do not manually edit this file.